### PR TITLE
Fix switch DB concurrent map issue

### PR DESCRIPTION
**Pull request by DivyaVavili**
Hit concurrent map issue with SwitchDB. Fixing it using concurrent-map

### CODE REVIEW

File: ofctrl/ofSwitch.go [hunk 1]
```diff
@@ -37,14 +38,18 @@ type OFSwitch struct {
...
+	switchDb = cmap.New()
+}
 
 // Builds and populates a Switch struct then starts listening
 // for OpenFlow messages on conn.
 func NewSwitch(stream *util.MessageStream, dpid net.HardwareAddr, app AppInterface) *OFSwitch {
 	var s *OFSwitch
 
-	if switchDb[dpid.String()] == nil {
+	if Switch(dpid) == nil {
```
Comment (jojimt) [inline ofctrl/ofSwitch.go]: nit: Could rename Switch to getSwitch?

Comment (DivyaVavili) [inline ofctrl/ofSwitch.go]: Will do

**Comment by abhinandanpb** [comment]
LGTM

**Comment by rchirakk** [comment]
@DivyaVavili  can you godep update netplugin ?

**Comment by DivyaVavili** [comment]
@rchirakk : Are you hitting this issue? If so, I can update it right away. Otherwise, there is a PR open that would eventually update it.

**Comment by rchirakk** [comment]
@DivyaVavili, sure I can wait for your PR.

**Pull request merged**